tests/entities_column_value_simple_tests.py

```diff
@@ -103,8 +103,92 @@ def test_should_throw_exception_when_setting_an_invalid_value():
     eq_(column_value.value, str(text))
 
 
+def test_should_create_a_column_value_with_no_API_input_data():
```

Comment:
Please rename to test_should_create_a_people_column_value_with_no_api_input_data.  Moving forward, please always indicate the type of column value that you are testing as to not duplicate names and confuse the test runner.